os/binfmt/libelf/libelf_load.c

```diff
@@ -352,6 +352,23 @@ int elf_load(FAR struct elf_loadinfo_s *loadinfo)
 	}
 #endif
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+	/* Re-initialize the binary heap information.
+	 * Because, binary heap contains text and data region size,
+	 * but those should not be calculated for heap usage.
+	 */
+	pid_t pid = getpid();
+	pid_t hashpid = PIDHASH(pid);
+
```

Comment:
uheap is under "CONFIG_APP_BINARY_SEPARATION". Should we add dependency here?

Reply:
I added. Thanks.